extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -45,6 +45,9 @@ class AmpStickyAd extends AMP.BaseElement {
 
     /** @private {?UnlistenDef} */
     this.scrollUnlisten_ = null;
+
+    /** @private {boolean} */
+    this.collapsed_ = false;
```

Comment:
given collapsedCallback is in custom-element, shall we have the state there?

Reply:
Can't really do that. sticky-ad is a little bit different because we always collapse it by setting its display to none. For other components, collapsedCallback is more like a attemptToCollapseCallback, which I can't simply set `this.collapsed_` to true.